extras/README.md

```diff
@@ -7,24 +7,26 @@ This document assumes you have `docker` installed.
 ## Download the latest cord container image (or you can choose to build locally)
 
 ```sh
-docker pull dhiway/cord:develop
+docker pull dhiway/cord
 ```
 
-## Generate `node-key`
+## Generate `node.key`
 
 In a blockchain, every participating node should be uniquely identified. Hence, one needs to generate a new key for the node, which needn't be shared with anyone.
+```sh
+sudo mkdir /tmp/cord && sudo chown $USER /tmp/cord && sudo chmod 777 /tmp/cord
```

Comment:
How about simplifying the directory handling by having volume details handled by Docker? This requires changes to the docker image.

Reply:
In order for docker to mount a volume in local machine, we need a local directory. But if we want to use container filesystem itself and not to mount at all, then upon removing the container, we'll loose data. But here we need the node key.

Reply:
@roynrishingha  checked with Satish - this comment is not a blocker, I'll approve this change if you have tested this  to be working fine.
For handling directory, lets open another task and discuss on a good way to do it..